### ISSUE TITLE
ogg files not working

### ISSUE THREAD

**Issue by KMonster5123** [leinelissen/jellyfin-audio-player#97]
I have a few songs in the Ogg format and its a hassle to convert them to FLAC and transfer the metadata
They technically do work but they worked in the sense that FLAC files did as they don't show the correct track length (It only shows as much as it loads and although it still plays is stuck on 0:00) and the next song is paused and doesn't play automatically when the previous song finishes
This is the case on ios and on android it just refuses to play

**Comment by leinelissen** [comment]
Hey, thanks for the feedback! The problem with OGG is that it's not supported natively on iOS, and thus the Jellyfin server must convert the stream in real-time to a format that iOS can actually playback. While this process works mostly smoothly in the background, this does mean that iOS displays it as an actual stream (including the track length updating as the stream is pulled in). We can admittedly do a better job of showing this in the UX, but it's quite tied to the OS, so not necessarily straightforward to solve. 
However, Android does support OGG and it should be played back natively. I'll do a check to see what's going wrong here.

**Comment by Behem07h** [comment]
would it be possible to simply request the jellyfin server to remux the files into a .caf container? ios native player supports playing opus via .caf

**Comment by Behem07h** [comment]
i'm not sure what jellyfin's api is like but i can confirm that ffmpeg supports remuxing to caf containers

**Comment by leinelissen** [comment]
Yeah there's a thin layer between ffmpeg and the Jellyfin API that I've fought with before, I can check if they explicitly define caf as a container format.

**Comment by jmshrv** [comment]
I'm investigating remuxing/transcoding stuff to OPUS/CAF myself, it's technically possible via Jellyfin but the resulting audio file doesn't play because of how Jellyfin returns the transcoded file (see my rambling here - https://github.com/jmshrv/finamp/issues/215)

At some point I plan to fix this in Jellyfin, probably by adding a parameter to UniversalAudio to "wait" for a transcode to finish before serving the file.

**Comment by leinelissen** [comment]
Hey @jmshrv, thanks for looping me in and thanks for the awesome work. I'd love to see the toggle to wait for the transcode to finish because the iOS background player isn't too keen on playing back streams of audio and of course fixing OPUS playback. Big win for the ecosystem. I'll track your work and integrate it as soon as it's available.
